dotCMS/src/main/java/com/dotcms/rest/exception/mapper/ElasticsearchStatusExceptionMapper.java

```diff
@@ -0,0 +1,29 @@
+package com.dotcms.rest.exception.mapper;
+
+import com.dotmarketing.util.Logger;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import org.elasticsearch.ElasticsearchStatusException;
+import javax.ws.rs.ext.ExceptionMapper;
+
+public class ElasticsearchStatusExceptionMapper implements ExceptionMapper<ElasticsearchStatusException> {
+
+
+    @Override
+    public Response toResponse(ElasticsearchStatusException exception) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'exception' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5171086)